cypress/support/commands.ts

```diff
@@ -0,0 +1,125 @@
+import { LoginPagePo } from '@/cypress/e2e/po/pages/login-page.po';
```

Comment:
i think there's some merge issues with this file. it's showing up as new, but does exist in `master` at the moment https://github.com/rancher/dashboard/blob/master/cypress/support/commands/commands.ts

some commands have moved in to https://github.com/rancher/dashboard/blob/master/cypress/support/commands/rancher-api-commands.ts (like login), some have stayed byLabel, findId, getId, keyboardControls, interceptAllRequests and iFrame

Reply:
I derped. the new location is cypress/support/commands/commands.ts. If there are any changes in this PR to the file they need to be moved across